functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -58,8 +85,107 @@ def get_user_by_id(conn, id):
             columns = [column[0] for column in cursor.description]
             data = dict(zip(columns, row))
 
+            logging.debug("Users retrieved successfully!")
             return func.HttpResponse(
                 json.dumps(data),
                 status_code=200,
                 mimetype="application/json"
             )
+
+
+def post():
+    logging.debug("POST is not implemented")
+    return func.HttpResponse(
+        "Method not allowed!",
+        status_code=405
+    )
+
+
+def put(req, conn, id):
```

Comment:
Maybe the param should be user_id rather than id. I know that it's obvious in this file that we're only really using user ids but in general two different kinds of ids show up in the functions so there might still be some confusion.

Reply:
I updated it to user_id!